openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -189,7 +189,7 @@ Requirements for both the Wallet and the Verifier:
 * Response type MUST be `vp_token`.
 * Response mode MUST be `direct_post.jwt`. The Verifier MUST return `redirect_uri` in response to the HTTP POST request from the Wallet, where the Wallet redirects the User to, as defined in Section 8.2 of [@!OIDF.OID4VP]. Implementation considerations for the response mode `direct_post.jwt` are given in Section 14.3 of [@!OIDF.OID4VP].
 * Authorization Request MUST be sent using the `request_uri` parameter as defined in JWT-Secured Authorization Request (JAR) [@!RFC9101].
-* For Client Identifier Prefix as introduced in Section 5.10 of [@!OIDF.OID4VP], `x509_hash` MUST be supported by the Wallet and used by the Verifier.
+* For Client Identifier Prefix as introduced in Section 5.9.3 of [@!OIDF.OID4VP], `x509_hash` MUST be supported by the Wallet and used by the Verifier.
```

Comment:
The introduction of prefixes is actually in 5.9 - maybe it reads better like this?

```suggestion
* The Client Identifier Prefix `x509_hash` MUST be supported by the Wallet and used by the Verifier (see Section 5.9.3 of [@!OIDF.OID4VP])
```